doc/manual/manual.tex

```diff
@@ -5569,6 +5569,48 @@ \subsubsection{Prescribed internal velocity constraints}
     \label{fig:prescribed-velocity}
 \end{figure}
 
+\subsubsection{Prescribing internal velocity constraints with ASCII files}
+\label{sec:prescribed-velocities-ascii-data}
+\textit{This section was contributed by Bob Myhill}
+
+Building on \ref{sec:prescribed-velocities}, the \url{cookbooks/prescribed_velocity_ascii_data}
+directory contains a plugin which uses an ASCII data file to specify where to prescribe
+internal velocities. Velocities are prescribed wherever the field value indicated by the ASCII data file
+is less than 0.5. As before, the plugin is loaded in parameter files as an additional shared library:
```

Comment:
I *think* that in the code you actually use "greater than 0.5".

Reply:
Good catch!